### PR TITLE
feat(Unicon): add hover-fill property

### PR DESCRIPTION
**Pull request by E1mir**
Thank you for your contribution to the vue-unicons repo.
Before submitting this PR, please make sure:

- [X] Your code builds clean without any errors or warnings
- [x] Your commits follow the [сonvention](https://github.com/antonreshetov/vue-unicons/blob/master/.github/COMMIT_CONVENTION.md)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [X] Feature
- [ ] Refactor

**Importantly**

To avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on PR.

### CODE REVIEW

File: src/components/Unicon.vue [hunk 1]
```diff
@@ -66,17 +78,30 @@ export default {
...
+      }
+    }
   }
 }
 </script>
 
 <style>
 .unicon {
   display: inline-block;
+  transition: 0.3s all;
```
Comment (antonreshetov) [inline src/components/Unicon.vue:104]: Let's reduce it to 0.2.s, it's too long ). And by the way, as it turns out `fill: var(--uni-color-primary);` is not used, so you can remove that. To add a transition, you need to add it like this - `.unicon svg {transition: 0.2s all}`

File: tests/unit/Unicon.spec.js [hunk 2]
```diff
@@ -65,4 +65,28 @@ describe('Unicon', () => {
     svg.trigger('click')
     expect(wrapper.emitted().click).toBeTruthy()
   })
+  it('hover fill is working', () => {
+    const wrapper = shallowMount(Unicon, {
+      propsData: {
```
Comment (antonreshetov) [inline tests/unit/Unicon.spec.js:70]: You could simply extend `baseProps` rather than adding repeated code

**Review by antonreshetov — CHANGES_REQUESTED** [review]

File: tests/unit/Unicon.spec.js (hunk 2 above)
Comment (E1mir) [reply tests/unit/Unicon.spec.js:70]: Fixed

**Review by antonreshetov — APPROVED** [review]

**Comment by leyenda** [comment]
Hello, this PR broke the reactivity of the fill prop. Now we cant change the icon fill from the parent, after icon mounted.

**Comment by E1mir** [comment]
You mean, change via css or what? Could you show me where it happens, and I will try to fix that asap

**Comment by antonreshetov** [comment]
@leyenda It's not quite clear what we're talking about.
P.S. merged PR is not a good place for discussion. Please open an Issue on this subject and provide an example.

**Comment by leyenda** [comment]
Thank you for your responses, I'll open a new issue to make it more clear.

**Pull request merged**